app/views/_components/primary-navigation/template.njk

```diff
@@ -11,7 +11,7 @@
         <ul class="app-primary-navigation__list">
 
           {%- for item in params.items %}
-            <li class="app-primary-navigation__item">
+            <li class="app-primary-navigation__item {{item.classes}}">
```

Comment:
Will `item.classes` always exist? If not, I think it should be:

```
{{- ' ' + item.classes if item.classes }}
```
For example:

```
<li class="app-primary-navigation__item {{- ' ' + item.classes if item.classes }}">
```

Reply:
k